.github/workflows/tests-pro-integration.yml

```diff
@@ -339,7 +339,7 @@ jobs:
           AWS_DEFAULT_REGION: "us-east-1"
           JUNIT_REPORTS_FILE: "pytest-junit-community-${{ matrix.group }}.xml"
           TEST_PATH: "../../localstack/tests/aws/"  # TODO: run tests in tests/integration
-          PYTEST_ARGS: "${{ env.TINYBIRD_PYTEST_ARGS }}${{ env.TESTSELECTION_PYTEST_ARGS }}--splits ${{ strategy.job-total }} --group ${{ matrix.group }} --durations-path ../../localstack/.test_durations --store-durations"
+          PYTEST_ARGS: "${{ env.TINYBIRD_PYTEST_ARGS }}${{ env.TESTSELECTION_PYTEST_ARGS }}--splits ${{ strategy.job-total }} --group ${{ matrix.group }} --durations-path ../../localstack/.test_durations --store-durations --ignore ../../localstack/tests/aws/services/cloudformation/v2"
```

Comment:
FYI: As far as I can tell, this change really just migrates the ignore introduced in https://github.com/localstack/localstack/pull/12871 to the tests executed against Pro.